ui/apps/platform/src/Containers/VulnMgmt/RiskAcceptance/useImageVulnerabilities.ts

```diff
@@ -3,30 +3,36 @@ import {
     GetImageVulnerabilitiesData,
     GetImageVulnerabilitiesVars,
     GET_IMAGE_VULNERABILITIES,
+    GET_IMAGE_VULNERABILITIES_LEGACY,
 } from './imageVulnerabilities.graphql';
 
-function useImageVulnerabilities({ imageId, vulnsQuery, pagination }) {
+function useImageVulnerabilities({ imageId, vulnsQuery, pagination, showVMUpdates = false }) {
     const client = useApolloClient();
+    const queryToUse = showVMUpdates ? GET_IMAGE_VULNERABILITIES : GET_IMAGE_VULNERABILITIES_LEGACY;
+
     const {
         loading: isLoading,
         data,
         error,
-    } = useQuery<GetImageVulnerabilitiesData, GetImageVulnerabilitiesVars>(
-        GET_IMAGE_VULNERABILITIES,
-        {
-            variables: {
-                imageId,
-                vulnsQuery,
-                pagination,
-            },
-            fetchPolicy: 'network-only',
-        }
-    );
+    } = useQuery<GetImageVulnerabilitiesData, GetImageVulnerabilitiesVars>(queryToUse, {
+        variables: {
+            imageId,
+            vulnsQuery,
+            pagination,
+        },
+        fetchPolicy: 'network-only',
+    });
 
     async function refetchQuery() {
-        await client.refetchQueries({
-            include: [GET_IMAGE_VULNERABILITIES],
-        });
+        if (showVMUpdates) {
```

Comment:
Does it work if you pass the `queryToUse` in the `includes` array instead of using a conditional here?
```
await client.refetchQueries({
    include: [queryToUse],
});
```